resources/styles/components/task-step/reading/book-content/index.less

```diff
@@ -0,0 +1,141 @@
+//
+// **NOTE:** This file must always be imported within a scope.
+// Otherwise, it will style all sorts of things like arbitrary lists and paras.
+//
+// TODO: Maybe this file should be moved out of components since it styles HTML not generated by React.
+//
+@physics-color-primary: #79b142;
+@physics-background-primary: #232e66;
+@physics-background-secondary: #79b142;
+// @tips-for-success-color:#bf007b;
+// @grasp-check-color:#39b531;
+// @grasp-check-color-background:#fff;
+
+@import './links-to-physics';
+@import './work-in-physics';
+@import './tips-for-success';
+
+
+// Hide CNX Processing instructions
+cnx-pi { display: none; }
+
+.learning-objectives {
+
+  #fonts > .sans(1.8rem, 1.8rem);
+
+  width: 100%;
+  min-height: 100px;
+  display: inline-flex;
+  margin-top: -20px;
+  color: #ffffff;
+  padding: 10px 40px;
+  background: @physics-background-secondary;
+
+  // TODO: What is this hiding?
+  > div { display: none; }
+
+  p {
+    color: rgba(255,255,255,.75);
+    #fonts > .sans();
+    text-transform: uppercase;
+    text-align: right;
+    margin: auto 15px;
+  }
+
+  ul {
+    margin: 0 0;
+    padding: 0 15px 0 30px;
+    border-left: 1px solid rgba(255,255,255,.4);
+    ul {
+      display: flex;
+      flex-direction: column;
+    }
+    .ost-learning-objective-def {
+      font-weight: 300;
+      font-size: 1.5rem;
+      list-style-type: none;
+      flex: 1;
+      display: flex;
+      justify-content: center;
+      flex-direction: column;
+    }
+  }
+
+}
+
+
+.section-opener ~ section h1 + p::first-letter,
+div[data-type="document-title"] ~ p::first-letter {
+  float: left;
+  #fonts > .sans(11rem, 9rem);
+  font-weight: 600;
+  color: @physics-color-primary;
+  margin: 0 4px 0 -6px;
+}
+
+div[data-type="document-title"] ~ p {padding: 0 140px;}
+
+
+> section:not(.section-opener),
+> section > section,
+.worked-example,
+.virtual-physics { padding: 0 140px; }
+
+> section { padding: 0; }
+
+.splash {
+  width: 100%;
+  margin: -20px 0 0 0;
```

Comment:
Rather than setting negative margins to compensate for padding on `.card-body`, what if `.card-body` had no margins and `.card-body:first()` had a `margin-top`?  Or, instead of using `20px` here we could refer to a variable containing the `.card-body` padding. Thoughts @Fredasaurus?